<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1093</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame3">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1111</width>
     <height>531</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTextEdit" name="FAPtextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>501</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(231, 231, 231);
color: rgb(255, 0, 0);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FAPpushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(135, 135, 135);</string>
    </property>
    <property name="text">
     <string>Prevent FakeAP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FinputTxt">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(231, 231, 231);
color: rgb(255, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>BSSID of FakeAP :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cantarell</family>
      <pointsize>25</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 25pt &quot;Cantarell&quot;;</string>
    </property>
    <property name="text">
     <string>INTRUSION PREVENTION</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="UMACtextEdit">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>300</y>
      <width>501</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(231, 231, 231);
color: rgb(255, 0, 0);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UMACpushButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>210</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(135, 135, 135);</string>
    </property>
    <property name="text">
     <string>Prevent UnauthMAC</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>130</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);;</string>
    </property>
    <property name="text">
     <string>BSSID of UnauthMAC :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="UinputTxt">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>130</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(231, 231, 231);
color: rgb(255, 0, 0);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_fkbutton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>220</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(135, 135, 135);</string>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_unbutton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>220</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(135, 135, 135);</string>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
